server/src/users/entities/user.entity.ts

```diff
@@ -1,6 +1,7 @@
 import { Entity, Column, PrimaryGeneratedColumn, OneToMany } from 'typeorm';
 import { Asset } from '../../assets/entities/asset.entity';
 import { Message } from '../../messages/entities/message.entity';
+import { Transaction } from '../../transactions/entities/transaction.entity'
```

Comment:
That throws the following error:

src/users/entities/user.entity.ts:26:20 - error TS1361: 'Transaction' cannot be used as a value because it was imported using 'import type'.

26   @OneToMany(() => Transaction, (transaction)=> transaction.donater_user)